src/views/News/index.js

```diff
@@ -297,10 +297,10 @@ export default class StudentNews extends Component {
     const networkStatus = JSON.parse(localStorage.getItem('network-status')) || 'online';
 
     if(this.props.Authentication) {
-
+      
       if (this.state.loading === true) {
         content = <GordonLoader />;
-      } else if (this.state.news.length > 0) {
+      } else if (this.state.news.length > 0 || this.state.personalUnapprovedNews.length > 0) {
```

Comment:
The name "personalUnapprovedNews" sounds like it's part of the student news. Were you tweaking something with the student news to implement something for ApartApp?

Reply:
That is not a change that I made, I believe it came in from merging changes that were part of the real develop branch